Resources/Private/Javascript/Connector.js

```diff
@@ -17,6 +17,24 @@ export class ShopwareConnector {
     });
   }
 
+  handleError(error) {
+    document.dispatchEvent(new CustomEvent('error', {
```

Comment:
'error' is really generic, should at least be prefixed to be identifiable as coming from this class